cvmfs/network/jobinfo.h

```diff
@@ -72,6 +72,9 @@ class JobInfo {
   unsigned backoff_ms_;
   unsigned int current_host_chain_index_;
 
+  // allow failure of downloads. default = false
```

Comment:
It's better to not repeat the code in the comment. Perhaps
```suggestion
  // Don't fail-over proxies on download errors. default = false
```

Reply:
Needs to be initialized in `Init()`.